tests/observatory_platform/cli/test_observatory.py

```diff
@@ -145,6 +144,7 @@ def test_platform_check_dependencies(self, mock_is_docker_running, mock_get_dock
 class TestObservatoryTerraform(unittest.TestCase):
     # TODO change to 'observatory-unit-testing' once latest version is available
     organisation = 'COKI-project'
+    workspaces_prefix = datetime.now().strftime("%Y-%M-%d_%H-%M-%S-%f")
```

Comment:
It might be a little safer to use a random uuid, it probably won't matter, but there is already a function `random_id` that will make one. You can import it with `from tests.observatory_platform.config import random_id`. It would at least make it consistent with the other tests that do similar things.